templates/publisher/market.html

```diff
@@ -1,11 +1,155 @@
 {% extends "_layout.html" %}
 
 {% block title %}
-  Market page — Linux software in the Snap Store
+  Market details for {{ snap_title }}
 {% endblock %}
 
 {% block content %}
+  <div id="main-content" class="u-no-margin--top">
+    <section class="p-strip is-shallow">
+      <div class="row">
+        <p><a href="/snaps">My snaps&nbsp;&rsaquo;</a></p>
+        <h1 class="u-no-margin--top">{{ snap_title }}</h1>
+      </div>
+    </section>
+    <section class="metrics__nav-wrap p-strip is-shallow u-no-padding--top u-no-padding--bottom">
+      <nav class="p-tabs">
+        <ul class="p-tabs__list" role="tablist">
+          <li class="p-tabs__item" role="presentation">
+            <a href="/account/snaps/{{ snap_name }}/measure" class="p-tabs__link" tabindex="0" role="tab" aria-controls="section1">Measure</a>
+          </li>
+          <li class="p-tabs__item" role="presentation">
+            <a href="/account/snaps/{{ snap_name }}/market" class="p-tabs__link" tabindex="0" role="tab" aria-controls="section2" aria-selected="true">Market</a>
+          </li>
+        </ul>
+      </nav>
+    </section>
 
-{{ metadata }}
+    <div class="p-strip is-shallow">
+      <form method="POST">
```

Comment:
add this hidden input:
`<input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>`